.github/workflows/ci.yaml

```diff
@@ -178,39 +178,51 @@ jobs:
             EXPERIMENTAL_FEATURES: 1
             TEST_DB_PROVIDER: sqlite3
             COMPILER: gcc
+            TEST_NETWORK: regtest
           - NAME:  gcc-dev1-exp0
             CFG: gcc-dev1-exp0
             DEVELOPER: 1
             EXPERIMENTAL_FEATURES: 0
             TEST_DB_PROVIDER: sqlite3
             COMPILER: gcc
+            TEST_NETWORK: regtest
           - NAME: gcc-dev0-exp1
             CFG: gcc-dev0-exp1
             DEVELOPER: 0
             EXPERIMENTAL_FEATURES: 1
             TEST_DB_PROVIDER: sqlite3
             COMPILER: gcc
+            TEST_NETWORK: regtest
           - NAME: gcc-dev0-exp0
             CFG: gcc-dev0-exp0
             DEVELOPER: 0
             EXPERIMENTAL_FEATURES: 0
             TEST_DB_PROVIDER: sqlite3
             COMPILER: gcc
+            TEST_NETWORK: regtest
           # While we're at it let's try to compile with clang
           - NAME: clang-dev1-exp1
             CFG: clang-dev1-exp1
             DEVELOPER: 1
             EXPERIMENTAL_FEATURES: 1
             TEST_DB_PROVIDER: sqlite3
             COMPILER: clang
+            TEST_NETWORK: regtest
           # And of course we want to test postgres too
           - NAME: postgres
             CFG: gcc-dev1-exp1
             DEVELOPER: 1
             EXPERIMENTAL_FEATURES: 1
             COMPILER: gcc
             TEST_DB_PROVIDER: postgres
-
+            TEST_NETWORK: regtest
+          # And don't forget about elements (like cdecker did when
+          # reworking the CI...)
+          - NAME: gcc-dev1-exp0
```

Comment:
maybe call this liquid-regtest then?

Reply:
That's what I get for "just doing this quickly before getting dinner" xD